src/math/Vector2.js

```diff
@@ -379,7 +372,13 @@ Object.assign( Vector2.prototype, {
 
 	normalize: function () {
 
-		return this.divideScalar( this.length() || 1 );
+		var l = this.length();
```

Comment:
TBH, the previous version looked much better. I would revert this change.

Reply:
Better but not faster.
There's a considerable cost on calling function vs. using expanded calculations.
I'd expand even .length line, but I kept as is because it is harder to mentally link expanded calculation to a length (missing the explanatory context).

I think calculation on a Vector2 is a priority, right?

Reply:
You should proof with a benchmark how much faster this calculation is. The the benefit is just minimal, I would keep the original version.

Reply:
Yes, I'll re-sort this.
Actually just found out the code is formatted using tabs with a size of 2, and not 2 spaces.
The download is taking a long, long, long time.

Would be nice if secondary resources got split into different org projects.